surveys/2022-annual-survey/questions.md

```diff
@@ -669,12 +596,6 @@ Aspects:
 - Disk space usage (e.g., the size of target folder)
 - Bugs in the compiler (i.e., ICEs a.k.a. internal compiler errors, miscompilations, etc.)
 - Compiler error messages
-- IDE experience
-- Debugging experience
-- Available tools and support
-- Async programming
-- GUI development
-- Rust language and standard library documentation
```

Comment:
Thinking more about it, I'm not sure about this...

Reply:
Note to self (from Wesley): could we get respondents to rank these in priority order, to discover how people would feel about sacrificing, e.g., disk space for compile times?

Reply:
I think having all of these rated would be useful.

Reply:
@nrc can you add these back, and then we can discuss separately whether to cut them?